firrtl/src/main/scala/firrtl/ir/IR.scala

```diff
@@ -257,6 +257,29 @@ case class PathPropertyLiteral(value: String) extends Expression with UseSeriali
 
 case class SequencePropertyValue(tpe: Type, values: Seq[Expression]) extends Expression with UseSerializer
 
+/** Property primitive operations.
+  */
+sealed trait PropPrimOp
+object PropPrimOp {
+  case object AddOp extends PropPrimOp {
+    override def toString(): String = "integer_add"
```

Comment:
```suggestion
    override def toString: String = "integer_add"
```
Nit but `()` imply mutation which is obviously not the case here.

You could also consider making `PropPrimOp` an `abstract class` that takes the String representation as an argument and overriding `toString` there so that future Ops are a little more concise to add, feel free to not doing this though.